server/templates/staff/sidebar.html

```diff
@@ -20,7 +20,7 @@
             {% for course in courses %}
               <li class="treeview  {% if (request.endpoint == "admin.index" and loop.length == 1) %}active default{% endif %} {% if request.view_args['cid'] == course.id %} active{% endif %}" >
                 <a href="#">
-                  <i class="fa fa-university"></i> <span>{{ course.display_name_with_semester }}</span>
+                  <i class="fa fa-university"></i> <span>{{ course.display_name_with_semester | truncate(25, True) }}</span>
```

Comment:
Does this line wrap if it's too long? Another option would be a text-overflow css property